From 401ffb64b38b9ad72629a897ca7fdc447858a3bd Mon Sep 17 00:00:00 2001
From: "Kweh, Hock Leong" <hock.leong.kweh@intel.com>
Date: Thu, 15 Sep 2016 23:47:29 +0800
Subject: [PATCH 11/11] samples/flow/iio: add iio-hid sensors json file

add a configuration file in JSON format for iio-hid sensors supported in
Linux mainline (v4.8):
Accelerometer, Gyrometer, Ambient Light, Magnetometer and Barometer.

Upstream-Status: Pending

Signed-off-by: Kweh, Hock Leong <hock.leong.kweh@intel.com>
---
 src/samples/flow/iio/sol-flow-hid.json |   53 ++++++++++++++++++++++++++++++++
 1 file changed, 53 insertions(+)
 create mode 100644 src/samples/flow/iio/sol-flow-hid.json

diff --git a/src/samples/flow/iio/sol-flow-hid.json b/src/samples/flow/iio/sol-flow-hid.json
new file mode 100644
index 0000000..629cf8a
--- /dev/null
+++ b/src/samples/flow/iio/sol-flow-hid.json
@@ -0,0 +1,53 @@
+{
+  "$schema": "http://solettaproject.github.io/soletta/schemas/config.schema",
+  "nodetypes": [
+    {
+      "name": "timer",
+      "type": "timer",
+      "options": {
+        "interval": 1000
+      }
+    },
+    {
+      "name": "MyAccelerate",
+      "type": "iio/accelerometer",
+      "options": {
+        "iio_device": "accel_3d",
+        "buffer_size": -1
+      }
+    },
+    {
+      "name": "MyGyroscope",
+      "type": "iio/gyroscope",
+      "options": {
+        "iio_device": "gyro_3d",
+        "buffer_size": -1
+      }
+    },
+    {
+      "name": "MyLight",
+      "type": "iio/light-sensor",
+      "options": {
+        "iio_device": "als",
+        "buffer_size": -1
+      }
+    },
+    {
+      "name": "MyMagnet",
+      "type": "iio/magnetometer",
+      "options": {
+        "iio_device": "magn_3d",
+        "buffer_size": -1
+      }
+    },
+    {
+      "name": "MyPressure",
+      "type": "iio/pressure-sensor",
+      "options": {
+        "iio_device": "press",
+        "buffer_size": -1
+      }
+    }
+  ]
+}
+
-- 
1.7.9.5

